ts/tests/blueprints/ember-cli-typescript-test.ts

```diff
@@ -206,9 +206,8 @@ describe('Acceptance: ember-cli-typescript generator', function() {
 
           const globalTypes = file('types/global.d.ts');
           expect(globalTypes).to.exist;
-          expect(globalTypes)
-            .to.include("declare module 'my-app/ui/components/*/template'")
-            .to.include(`  import { TemplateFactory } from 'htmlbars-inline-precompile';
+          expect(globalTypes).to.include("declare module 'my-app/ui/components/*/template'").to
+            .include(`  import { TemplateFactory } from 'htmlbars-inline-precompile';
```

Comment:
This one's even worse. Can we tweak the rules to allow this?

Reply:
I'll take a look. We definitely can't be the first ones to run into `.to.<some method>` as a thing to preserve with it.